.golangci.yml

```diff
@@ -7,17 +7,20 @@ linters-settings:
     disabled-checks:
       - regexpMust
   goheader:
+    values:
+      regexp:
+        COMPANY: .*
     template: |-
-      Copyright © {{ YEAR }} Kaleido, Inc.
-      
+      Copyright © {{ YEAR }} {{ COMPANY }}
+
       SPDX-License-Identifier: Apache-2.0
-      
+
       Licensed under the Apache License, Version 2.0 (the "License");
       you may not use this file except in compliance with the License.
       You may obtain a copy of the License at
-      
```

Comment:
Don't think you can change the whitespace here, without flagging a diff on every file